### PR TITLE
[8.1.x] ISPN-6857 OutdatedTopologyException in clustered invalidation cache

### PR DESCRIPTION
**Pull request by panossot**
https://issues.jboss.org/browse/ISPN-6857

### CODE REVIEW

File: core/src/main/java/org/infinispan/util/logging/Log.java [hunk 1]
```diff
@@ -1416,5 +1416,8 @@ void preparedTxAlreadyExists(RecoveryAwareTransaction previous,
    @LogMessage(level = WARN)
    @Message(value = "Received unsolicited state from node %s for segment %d of cache %s", id = 396)
    void ignoringUnsolicitedState(Address node, int segment, String cacheName);
+
+   @Message(value = "Cannot enable '%s' in invalidation caches!", id = 397)
```
Comment (tristantarrant) [inline core/src/main/java/org/infinispan/util/logging/Log.java]: You cannot introduce a logging ID which conflicts with one used by future versions. You should use id 420.

**Comment by tristantarrant** [comment]
Also please amend the commit log to include the Jira.

File: core/src/main/java/org/infinispan/util/logging/Log.java (hunk 1 above)
Comment (panossot) [inline core/src/main/java/org/infinispan/util/logging/Log.java]: It should be ok now.

**Comment by tristantarrant** [comment]
Pushed to 8.1.x
Thanks

**Pull request merged**